File: backend/SocialNetwork.Repository/Errors/DirectMessageErrors.cs
namespace SocialNetwork.Repository.Errors
{
    public static class DirectMessageErrors
    {
        public const string SenderEmpty = "Sender ID cannot be empty.";
        public const string ReceiverEmpty = "Receiver ID cannot be empty.";
        public const string ContentEmpty = "Content cannot be empty.";
        public const string ContentTooLong = "Content cannot be longer than 280 characters.";
        public const string SenderDoesNotExist = "Sender does not exist.";
        public const string ReceiverDoesNotExist = "Receiver does not exist.";
    }
}

File: frontend/src/constants/directMessageErrors.ts
// DirectMessage error messages - should match backend DirectMessageErrors
export const DirectMessageErrors = {
  InvalidUserId: "Invalid or missing user id.",
  SenderEmpty: "Sender ID cannot be empty.",
  ReceiverEmpty: "Receiver ID cannot be empty.",
  ContentEmpty: "Content cannot be empty.",
  ContentTooLong: "Content cannot be longer than 280 characters.",
  SenderDoesNotExist: "Sender does not exist.",
  ReceiverDoesNotExist: "Receiver does not exist.",
} as const;
